ymcatwincities/openy
# [OYPD-187] PHP 7 and jquery_colorpicker

JIRA: https://propeople-us.atlassian.net/browse/OYPD-187

Steps for review:
- Login as admin
- Go to "/admin/structure/taxonomy/manage/color/add" and make sure you're seeing color widget.
